packages/okta-adapter/src/definitions/deploy/deploy.ts

```diff
@@ -64,6 +65,70 @@ const createCustomizations = (): Record<string, InstanceDeployApiDefinitions> =>
   })
 
   const customDefinitions: Record<string, Partial<InstanceDeployApiDefinitions>> = {
+    [BRAND_THEME_TYPE_NAME]: {
+      requestsByAction: {
+        default: {
+          request: {
+            transformation: {
+              omit: [ID_FIELD, LINKS_FIELD],
+            },
+          },
+        },
+        customizations: {
+          add: [
+            {
+              // BrandThemes are created automatically by Okta when a Brand is
+              // created, but we're allowing to "add" a theme alongside its
+              // Brand. We first send a GET query to get its ID, and then we
+              // perform a "modify" action.
```

Comment:
super nit - was this formatted this way? otherwise my OCD instincts will appreciate this 😅 

```suggestion
              // BrandThemes are created automatically by Okta when a Brand is created,
              // but we're allowing to "add" a theme alongside its Brand.
              // We first send a GET query to get its ID, and then we perform a "modify" action.
```

Reply:
I'm used to wrap comments at 80 characters :/